src/repl.ts

```diff
@@ -302,7 +299,7 @@ export interface TextDocumentPositionParams {
     position: vscode.Position
 }
 
-let getBlockText = new rpc.RequestType<TextDocumentPositionParams, void, void, void>('julia/getCurrentBlockText')
+let getBlockText = new rpc.RequestType<TextDocumentPositionParams, void, void, void>('julia/getCurrentBlockOffsetRange')
```

Comment:
Could we actually delete the whole line? ``getBlockText`` doesn't seem to be used anywhere. If so, lets delete it for v0.9.0, doesn't do any harm for now.